pxr/imaging/plugin/hdRpr/renderBuffer.cpp

```diff
@@ -124,6 +130,7 @@ VtValue HdRprRenderBuffer::GetResource(bool multiSampled) const {
         VtDictionary dictionary;
         dictionary["is_hybrid"] = isHybrid;
         dictionary["framebuffer"] = color;
+        dictionary["mutex_ptr"] = &color->GetContext().GetMutex();
```

Comment:
As far as I understood, you are using this mutex for two purposes:
1. Protect RPR API access [here](https://github.com/Radeon-Pro/RPRViewer/blob/99d9c855220bdea896bebc3728d56025d2c49e6e/RPRViewer/src/hdx/rprInteropPresenter.cpp#L58)
2. Protect access to `_presentedCondition`

This mutex will work for the second purpose but it will not work for the first one. This mutex does not actually protect RPR API calls because hdRpr uses another mutex for this purpose. hdRpr uses [this](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/master/pxr/imaging/plugin/hdRpr/rprApi.cpp#L79) mutex.